dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/CompanyDataControllerGetCompaniesEndpointTest.kt

```diff
@@ -1,10 +1,6 @@
 package org.dataland.e2etests.tests
 
-import org.dataland.datalandbackend.openApiClient.model.CompanyInformation
-import org.dataland.datalandbackend.openApiClient.model.DataMetaInformation
-import org.dataland.datalandbackend.openApiClient.model.IdentifierType
-import org.dataland.datalandbackend.openApiClient.model.QaStatus
-import org.dataland.datalandbackend.openApiClient.model.StoredCompany
+import org.dataland.datalandbackend.openApiClient.model.*
```

Comment:
Is the linter fine with this?

Reply:
It should not be but since this file lies not in in the frontend, the linter does not "see" it. 
I changed my IDE settings and now these wildcard-imports are not possible anymore.
Also changed this import.

Reply:
ok